src/Network/Network/ChangeLog.md

```diff
@@ -25,6 +25,7 @@
 * Fixed the incorrect type of `-TotalBytesPerSession` in `New-AzNetworkWatcherPacketCapture`
 * Added support for connection flushing in network security group which when enabled, re-evaluates flows when rules are updated
     - `New-AzNetworkSecurityGroup`
+* Add support for state in WAF Custom Rule
```

Comment:
Use past tense for verb `Add` in change log

Reply:
Done